beacon-chain/core/altair/epoch_precompute.go

```diff
@@ -98,12 +98,7 @@ func ProcessInactivityScores(
 		if v.IsPrevEpochTargetAttester && !v.IsSlashed {
 			// Decrease inactivity score when validator gets target correct.
 			if v.InactivityScore > 0 {
-				score := uint64(1)
-				// Prevents underflow below 0.
-				if score > v.InactivityScore {
-					score = v.InactivityScore
-				}
-				v.InactivityScore -= score
+				v.InactivityScore -= 1
```

Comment:
looking here: https://github.com/ethereum/annotated-spec/blob/master/altair/beacon-chain.md#inactivity-scores

I guess the `-= min(1, state.inactivity_scores[index])` is just trying to prevent the number from going negative (ie `state.inactivity_scores[index]` will either be >= 1 or 0)